alb.tf

```diff
@@ -67,17 +67,17 @@ resource "aws_alb_listener" "http" {
 resource "aws_alb_target_group" "default" {
   count       = local.load_balancer_count
   name        = var.name
-  port        = 80
-  protocol    = "HTTP"
+  port        = var.target_port
+  protocol    = var.target_protocol
   target_type = "ip"
   vpc_id      = var.vpc_id
   tags        = var.tags
 
   health_check {
     interval            = 30
     timeout             = 3
-    path                = var.health_check_path
-    protocol            = "HTTP"
+    protocol            = var.target_protocol
+    path                = var.target_protocol != "TCP" ? var.health_check_path : null
```

Comment:
Should this maybe be: `var.target_protocol == "HTTP" ? null : var.health_check_path` Or are the only target options `TCP` and `HTTP`?